cmd/watcher/main.go

```diff
@@ -68,6 +68,7 @@ var (
 	namespace               = flag.String("namespace", corev1.NamespaceAll, "Should the Watcher only watch a single namespace, then this value needs to be set to the namespace name otherwise leave it empty.")
 	checkOwner              = flag.Bool("check_owner", true, "If enabled, owner references will be checked while deleting objects")
 	updateLogTimeout        = flag.Duration("update_log_timeout", 30*time.Second, "How log the Watcher waits for the UpdateLog operation for storing logs to complete before it aborts.")
+	storeEvent              = flag.Bool("store_event", true, "If enabled, events related to runs will also be stored")
```

Comment:
should be off by default

events are even more numerous than secrets and config map on a k8s cluster - we should not impact performance by default

Reply:
It should be false. It was enabled for testing. Will change it.

Reply:
bump @khrm  - you still need to change this default

Reply:
+1 should be off by default

Reply:
I have disabled this by default now.